pegasus/sites.v3/hourofcode.com/views/home_highlights.haml

```diff
@@ -3,11 +3,9 @@
     .col-md-8
       %br/
       %h2
-        Hour of Code
-        = campaign_date('year').to_i - 1
```

Comment:
Ah, so we were showing previous year on purpose?

Reply:
Yep. We were originally going to change it to 2018 when we flipped to `actual-hoc`. Then we would need to make sure to keep it 2018 for `post-hoc` in December, but change it back to current_year - 1 in January. Which all seemed too complex for what this achieves, so simplifying by removing the year.